spec/models/character_spec.rb

```diff
@@ -18,21 +18,79 @@
   end
 
   describe '#take_damage' do
```

Comment:
Let's add a `before { subject.update(health: 100) }` here. I know the factory sets it up like that, but I don't like depending on the data from the factory.